ipaserver/install/server/__init__.py

```diff
@@ -335,6 +335,14 @@ def dirsrv_config_file(self, value):
     def __init__(self, **kwargs):
         super(ServerInstallInterface, self).__init__(**kwargs)
 
+        if self.domain_level == constants.DOMAIN_LEVEL_0:
+            if (self.no_pkinit or self.pkinit_cert_files is not None or
```

Comment:
But it is contradicting, user specifies `--no-pkinit` and got error `pkinit on domain level 0 is not supported`